### ISSUE TITLE
Setup CI

### ISSUE THREAD

**Issue by c3st7n** [adamrodger/gcloud-ctx#9]
Would be good to have automated builds using Travis or similar for stable and nightly rust versions.

**Comment by adamrodger** [comment]
Yep 👍 There's a TODO in the README about setting up GitHub Actions to pre-build release binaries for the various OSs and attach them to a GitHub release automatically. I've looked at what `tokei` and `ripgrep` do, so I'd just rip that off 😀 

The same approach can be done for running the tests. Well, once #2 is done anyway and there are actually some worth running

**Comment by adamrodger** [comment]
Added simple CI script in https://github.com/adamrodger/gcloud-ctx/commit/f030320d3ff590dd7ba1cce752adeb50aa5d65ac

I'll raise a separate issue for another Actions file for creating releases automatically
